### DISCUSSION TITLE
Support the Slack Block Kit

### DISCUSSION THREAD

**Discussion by NickLarsenNZ** [grafana/grafana#64094] [Alerting]
Currently, Slack messages are quite crude and could look a little nicer by using the Block Kit.

For example, as shown on: https://hodovi.cc/blog/creating-awesome-alertmanager-templates-for-slack/, alert-manager was able to add actions, which I think look a lot cleaner than a stream of hyperlinks:

<img width="600" alt="image" src="https://user-images.githubusercontent.com/10092581/222642930-44769034-6f6a-4218-b9c9-cbdc603c4252.png">

More detail on the payload: https://api.slack.com/messaging/composing/layouts
Block Kit: https://api.slack.com/reference/block-kit/blocks#actions

_It looks like blocks are being used for the mentions: https://github.com/grafana/grafana/blob/8ea71d37c233ab927b6ea062435bc410af1ac32b/pkg/services/alerting/notifiers/slack.go#L267-L278_

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82843. Feel free to continue the discussion around this there. Thank you!
